inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.hpp

```diff
@@ -164,6 +165,11 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
         return val;
     }
 
+    static ngraph::bfloat16 ie_abs(const ngraph::bfloat16& val) {
```

Comment:
Looks like, it should be moved to data_utils

Reply:
This function, along with similar template functions, has been moved to data_utils in ff00896e4f32744195fd0f021ebb9b9e4ae2c5cd.